server/serverRun.py

```diff
@@ -261,55 +254,36 @@ def not_logged_in():
     return 'Not logged in!'
 
 
-@app.route('/signup.html', methods=['GET'])
-@logged_out()
-def signup_page():
-    return render_page('signup.html')
+# @app.route('/main.html', methods=['GET'])
```

Comment:
Forget to delete this?

Reply:
Talked in mumble; keeping to possibly use later as a API endpoint.

Reply:
Keeping this as it will be turned into an endpoint sometime in the near future, and this code can be recycled.